README.md

```diff
@@ -38,6 +38,28 @@ A: No. You can, but running `bundle exec zeus` instead of `zeus` adds precious s
 It is common to see tests running twice when starting out with Zeus. If you see your tests/specs running twice, you should try disabling `require 'rspec/autotest'` and `require 'rspec/autorun'` (for RSpec), or `require 'minitest/autorun'` (for Minitest). (see [#134](https://github.com/burke/zeus/issues/134) for more information).
 
 
+## Set up 
```

Comment:
Maybe worth pointing out we are setting up a railsapp with zeus here? After all it is possible to use zeus with "non" rails stuff (https://github.com/sideshowcoder/zeus-without-rails)